components/cloud-foundry/frontend/test/e2e/cf.organizations.spaces.spec.js

```diff
@@ -15,7 +15,7 @@
   var Q = require('q');
   var _ = require('lodash');
 
-  describe('CF - Manage Organizations and Spaces', function () {
+  fdescribe('CF - Manage Organizations and Spaces', function () {
```

Comment:
**You've got an fdescribe - so all the tests bar 1 are being skipped - I think there are quite a few failures when you undo this.**